/**:
  ros__parameters:
    max_linear_speed: 1.5          # m/s
    max_rotational_speed: 1.5      # rad/s
    linear_step: 0.1               # m/s per keypress
    rotational_step: 0.1           # rad/s per keypress
    publish_rate_hz: 20.0          # optional